src/sql/plot.py

```diff
@@ -247,6 +247,7 @@ def boxplot(
         ax.bxp(stats, vert=vert)
         ax.set_title(f"{column!r} from {table!r}")
         set_label(column)
+        print(column)
```

Comment:
please double check your PRs before requesting a review. this print statement looks like it was used for debugging

Reply:
On it. will be more thorough in the future